packages/vuetify/src/components/VTimePicker/VTimePicker.js

```diff
@@ -13,6 +13,8 @@ const rangeHours24 = createRange(24)
 const rangeHours12am = createRange(12)
 const rangeHours12pm = rangeHours12am.map(v => v + 12)
 const rangeMinutes = createRange(60)
+const rangeSeconds = createRange(60)
```

Comment:
it's same like `rangeMinutes` so you can use it (i'd say no need to rename it as the related code needs to be cleaned up anyway, but you can change it to `range60` or whatever if you want)

Reply:
ready